Mathlib/Probability/Independence/Basic.lean

```diff
@@ -816,15 +852,9 @@ lemma iIndepFun.indepFun_prodMk_prodMk₀ (h_indep : iIndepFun f μ) (hf : ∀ i
 
 variable {ι' : Type*} {α : ι → Type*} [∀ i, MeasurableSpace (α i)]
 
-open Function in
 lemma iIndepFun.precomp {g : ι' → ι} (hg : g.Injective) (h : iIndepFun f μ) :
```

Comment:
Could you move this lemma and the other one next to the lemmas you're adding? Better have a coherent organization in the file, parallel to the one in the kernel file.